src/quo/components/colors/color_picker/view.cljs

```diff
@@ -27,14 +27,18 @@
   (let [selected              (reagent/atom default-selected)
         {window-width :width} (rn/get-window)
         ref                   (atom nil)]
-    (rn/use-effect #(js/setTimeout (fn []
-                                     (.scrollToIndex ^js @ref
-                                                     #js
-                                                      {:animated     true
-                                                       :index        (.indexOf colors/account-colors
-                                                                               default-selected)
-                                                       :viewPosition 0.5}))
-                                   50))
+    (rn/use-effect
+     (fn []
+       (js/setTimeout
+        (fn []
+          (let [index (.indexOf colors/account-colors default-selected)]
+            (when (and @ref (>= index 0))
+              (some-> ^js @ref
```

Comment:
FYI: The component test caught two potential bugs here, 1st that the index could be -1 and then `scrollToIndex` would throw an exception, and 2nd `@ref` could be nil and then an exception would be thrown as well.